ElementX/Sources/Screens/AnalyticsSettings/View/AnalyticsSettingsScreen.swift

```diff
@@ -0,0 +1,57 @@
+//
+// Copyright 2022 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import SwiftUI
+
+struct AnalyticsSettingsScreen: View {
+    @ObservedObject var context: AnalyticsSettingsScreenViewModel.Context
+    
+    var body: some View {
+        Form {
+            analyticsSection
+        }
+        .scrollContentBackground(.hidden)
+        .background(Color.element.formBackground.ignoresSafeArea())
```

Comment:
This should probably use the style from compound to match the Settings screen (we're mid way through adopting these but the Figma design system hasn't caught up yet).
```suggestion
        .compoundForm()
```

Reply:
👍